### DISCUSSION TITLE
Adding Analyst Users to Local Deployment

### DISCUSSION THREAD

**Discussion by gokalpcelik** [broadinstitute/seqr#2805] [Q&A] [answered]
Hi. 

Can anyone guide me through a proper way to add analyst users to a local deployment of SEQR?
I don't remember seeing anything that tells exactly how its done. 

Regards.

**Comment by hanars** [comment, marked as answer]
If you want analysts in seqr, do the following:

-  go to the `/admin/auth/group/add/` page and create a user group named something descriptive like "analysts". It does not need any permissions
- for any user you want to be an analyst, go to the `admin/auth/user/`, select the user, add the "analysts" group to that user, and click save
- set the `ANALYST_USER_GROUP` environment variable in your seqr deployment to "analysts". If you are using the docker-compose deployment this would involve adding the following to the `environment` section of the `seqr` configuration:
```
- ANALYST_USER_GROUP=analysts
```

**Reply by gokalpcelik** [reply]
Awesome. The dashboard is quite lovely.
